tests/Builders/ComponentParameterBuilderTests.cs

```diff
@@ -0,0 +1,326 @@
+﻿using System;
+using System.Threading.Tasks;
+using Bunit.SampleComponents;
+using Microsoft.AspNetCore.Components;
+using Shouldly;
+using Xunit;
+
+namespace Bunit
+{
+    public class ComponentParameterBuilderTests
+    {
+        private readonly ComponentParameterBuilder<AllTypesOfParams<string>> _sut;
+
+        public ComponentParameterBuilderTests()
+        {
+            _sut = new ComponentParameterBuilder<AllTypesOfParams<string>>();
+        }
+
+        [Fact(DisplayName = "Add CascadingParameter (nullable integer) and Build")]
+        public void Test001()
+        {
+            // Arrange
+            const int value = 42;
+
+            // Arrange
+            _sut.Add(c => c.NamedCascadingValue, value);
+            var result = _sut.Build();
+
+            // Assert
+            result.Count.ShouldBe(1);
+
+            var parameter = result[0];
+            parameter.IsCascadingValue.ShouldBeTrue();
+            parameter.Name.ShouldBe(nameof(AllTypesOfParams<string>.NamedCascadingValue));
+            parameter.Value.ShouldBe(value);
+        }
+
+        [Theory(DisplayName = "Add Parameter (string) and Build")]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData("foo")]
+        public void Test002(string? value)
+        {
+            // Arrange and Act
+            _sut.Add(c => c.RegularParam, value);
+            var result = _sut.Build();
+
+            // Assert
+            result.Count.ShouldBe(1);
+
+            var parameter = result[0];
+            parameter.IsCascadingValue.ShouldBeFalse();
+            parameter.Name.ShouldBe(nameof(AllTypesOfParams<string>.RegularParam));
+            parameter.Value.ShouldBe(value);
+        }
+
+        [Fact(DisplayName = "Add Parameter (RenderFragment) and Build")]

```

Comment:
```suggestion
        [Fact(DisplayName = "Add a markup string to a RenderFragment parameter")]
```
More explicit description of tests. Please make similar changes to other test methods.